config/repositories.yaml

```diff
@@ -51,7 +51,7 @@ buildConfiguration:
     #  - A specific release build: 0.0.1 (without the "v" prefix)
     #  - The most recent build from the master branch: master
     - name: NNFMFU_VERSION
-      value: 0.1.0
+      value: master
```

Comment:
Will your tool catch this when it's time to release?

Reply:
It does have a piece that checks this.